add_sub.java

```diff
@@ -17,7 +17,10 @@ public static void main(String[] args) {
 		int answer = a + b;
```

Comment:
Rename "answer" variable as "add_answer".

Reply:
Okay, will make changes and send a pr